### PR TITLE
Update to upstream version 2.8.81 and added missing folder 

### PR DESCRIPTION
**Pull request by xgcssch**
Now using upstream version 2.8.81
Copy import directory to the share folder, so it can be used by clients.

### CODE REVIEW

File: sh.exe.stackdump [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Stack trace:
```
Comment (Rastaban) [inline sh.exe.stackdump]: was this file intended :)

Comment (xgcssch) [reply sh.exe.stackdump]: Oops. That slipped my attention. Will resolve it when i‘m Home.

**Comment by Rastaban** [comment]
Thanks for the port, this looks good to me.  After you get a chance to remove the extra file we should be able to merge it.
